src/sample_submission_data.R

```diff
@@ -0,0 +1,9 @@
+## Script that generates sample submission data to pass validation checks
+require(arrow) # for parquet writing
+source('sim_model_output.R')
```

Comment:
```suggestion
source(here::here('src/sim_model_output.R'))
```

Reply:
Cool, thanks for the suggestions! I've not seen the `here::here` before, learn new things every day :)